discovery/chan_series.go

```diff
@@ -89,23 +146,7 @@ func (c *chanSeries) UpdatesInHorizon(chain chainhash.Hash,
 		return nil, err
 	}
 	for _, nodeAnn := range nodeAnnsInHorizon {
-		// Ensure we only forward nodes that are publicly advertised to
```

Comment:
New diff removes the protection against leaking out nodes with purely private channels.

Reply:
good catch! filtering has been restored